patches/@openeth+truffle-typings+0.0.6.patch

```diff
@@ -1,8 +1,8 @@
 diff --git a/node_modules/@openeth/truffle-typings/index.d.ts b/node_modules/@openeth/truffle-typings/index.d.ts
-index 5de91fb..86d9264 100644
+index 5de91fb..85cf390 100644
 --- a/node_modules/@openeth/truffle-typings/index.d.ts
 +++ b/node_modules/@openeth/truffle-typings/index.d.ts
-@@ -46,11 +46,14 @@ declare namespace Truffle {
+@@ -46,11 +46,15 @@ declare namespace Truffle {
```

Comment:
were the changes `86d9264` → `85cf390` and `a` → `b` automatically generated by something? if not, how did you come up with this manual change?

Reply:
The steps required to patch a package is:
1. change the file in the `node_modules/<package-to-patch>` folder
2. run the command: `npx patch-package <package-to-patch>`

This file must be re-generated with the command at point 2. otherwise, if not modified properly, the patch file is invalid.

Reply:
could you please describe what changes took place for step 1 to generate this file?

Reply:
Sure, in our specific case, the file changed the same that was already modified by the patch and it is described at line #4, `node_modules/@openeth/truffle-typings/index.d.ts`. Make me clarify a bit more:
1. The package `@openeth/truffle-typings` is installed/generated when I run `npm install`
2. Then I changed the file `node_modules/@openeth/truffle-typings/index.d.ts` according to our needs, adding the `collectorContract` field where required.
3. I run the command `npx patch-package @openeth/truffle-typings` to generate the patch file, hence the file `patches/@openeth+truffle-typings+0.0.6.patch` was updated.

I hope it explains the steps I did, otherwise, please let me know, I can show them to you in a call.